pkg/model/model.go

```diff
@@ -160,6 +163,9 @@ type Manifest struct {
 	Dependencies IstioDependencies `json:"dependencies"`
 	// Version specifies what version of Istio this release is
 	Version string `json:"version"`
+	// EOLDate specifies what is the date when this Istio version become EOL
+	// Should be in format "yyyy-mm"
+	EOLDate string `json:"eolDate"`
```

Comment:
If we already have to update istioctl binary directly it seems like we could just update the helm notes directly as well? We can have a test to check they are in sync if needed

Will defer to @ericvn

Reply:
do you mean to update the helm notes directly on istio/istio repository?